docs/references/nextjs/auth-middleware.mdx

```diff
@@ -118,10 +118,10 @@ The `authMiddleware()` method accepts an optional object. The following options
 | --- | --- | --- |
 | `afterAuth?` | `function` | Called after the authentication middleware is executed. This function has access to the [`Authentication`](/docs/references/nextjs/authentication-object) object and can be used to execute logic based on the auth state. |
 | `apiRoutes?` | `string[]` | A list of routes that should return 401. You can use glob patterns to match multiple routes or a function to match against the request object. For example: `['/foo', '/bar(.*)']` or [/^\/foo\/.*$/] |
-| `audience?` | `string \| string[]` | A string or list of audiences. |
+| `audience?` | `string \| string[]` | A string or list of [audiences](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3). If passed, it is checked against the `aud` claim in the token. |
 | `authorizedParties?` | `string[]` | An allowlist of origins to verify against, to protect your application from the subdomain cookie leaking attack.<br/>For example:<br/>`['http://localhost:3000', 'https://example.com']`<br/>For more information, refer to the [reference guide](/docs/references/nodejs/token-verification#validate-the-authorized-party-of-a-session-token). |
 | `beforeAuth?` | `function` | A function called before the authentication middleware is executed. If a redirect response is returned, the middleware will respect it and redirect the user. If `false` is returned, the auth middleware will not execute and the request will be handled as if the auth middleware was not present. |
-| `clockSkewInSeconds?` | `number` | Overrides the default allowed clock skew. |
+| `clockSkewInSeconds?` | `number` | Overrides the default allowed clock skew. Clock skew specifies the allowed time difference (in seconds) between the server and client clocks when validating a token. |
```

Comment:
The `clockSkewInSeconds` is used in the token verification used in `server` so it's more like:

> specifies the allowed time difference (in seconds) between the Clerk server (generating the token) and the customer application server clocks when validating a token. 

I don't suggest the above phrase, but I would suggest we use something else than `client` since it may result in misunderstanding that clock skew is related to the end-user browser clock.

Reply:
Updated the copy - let me know how you feel about it !